ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<p>"that you can use in the pipeline definition" -&gt; "that are used  in the pipeline definition"</p><p> </p><p>"it can be referred to as <code>hw_task.output"</code></p><p><code>To me, this implies it can be named something else?</code></p><p> </p><p>"Note that in the" -&gt;</p><p>Note: In the"</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>